### PR TITLE
More docs community

### PR DESCRIPTION
**Pull request by carlopi**
Minor fixes, and expanded on descriptor.yml fields and how documentation gets generated.

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -1,22 +1,68 @@
...
+```sql
+CREATE TABLE functions_pre AS SELECT ... FROM duckdb_functions();
+LOAD extension_name;
+CREATE TABLE functions_post AS SELECT ... FROM duckdb_functions();
+SELECT * FROM functions_pre EXCEPT (FROM functions_post) ORDER BY ...;
+```
+
+This works well for detecting new functions, functions overload, new settings, and new types.
+
+Detection of other changes to the system (e.g., whether an additional Parser or Optimizer callback has been register) are viable, but not yet implemented. Those are reccomended to be provided as part of the `docs.extended_description` field.
```
Comment (samansmink) [inline documentation.md:64]:
```suggestion
Detection of other changes to the system (e.g., whether an additional Parser or Optimizer callback has been register) are viable, but not yet implemented. Those are recommended to be provided as part of the `docs.extended_description` field.
```

File: documentation.md [hunk 2]
```diff
@@ -1,22 +1,68 @@
...
+
+### Hosted Extension Documentation Page 
+
+Each Community Extension has a documentation page at: `https://community-extension.duckdb.org/extensions/$extension_name.html`.
+
+For example, this is the page for [quack]({% link extensions/quack.md %}).
+
+Documentation pages are generated from the fields provided in the YAML descriptor file that is part of the [Community Extensions Repository](https://github.com/duckdb/community-extensions) and from the auto-detected changes that a given extensions introduces in DuckDB.
+
+#### Autodetected changes
```
Comment (samansmink) [inline documentation.md:51]: I think this header is not necessary? If you want to keep it i would suggest renaming it to 'documentation generation' and start with a sentence explaining that the section is going to explain how the documentation is generated

**Review by samansmink — COMMENTED** [review]
Looks good! added some nitpicks

File: documentation.md (hunk 2 above)
Comment (carlopi) [reply documentation.md:51]: Thanks!

**Pull request merged**